src/view/treeNodes/repositoryChangesNode.ts

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { DescriptionNode } from "./descriptionNode";
```

Comment:
some tslint warnings here about using `'` instead of `"`

Reply:
Fixed in the next PR: https://github.com/microsoft/vscode-pull-request-github/pull/2036